pkgs/top-level/all-packages.nix

```diff
@@ -19749,10 +19749,12 @@ in
 
   libpulseaudio = libpulseaudio-vanilla;
 
-  pulseeffects-pw = callPackage ../applications/audio/pulseeffects {
-    boost = boost172;
+  easyeffects = callPackage ../applications/audio/easyeffects {
+    glibmm = glibmm_2_68;
   };
 
+  pulseeffects-pw = lib.warn "pulseeffects-pw has been renamed to easyeffects." easyeffects; # Deprecated
```

Comment:
This should be moved to `aliases.nix`. Also, I am not sure warnings are supported in top-level attributes (due to nix-env).

Reply:
Updated.